src/main/java/org/pesmypetcare/webservice/entity/EventEntity.java

```diff
@@ -0,0 +1,87 @@
+package org.pesmypetcare.webservice.entity;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.EventDateTime;
+import com.google.api.services.calendar.model.EventReminder;
+import lombok.Data;
+import com.google.api.services.calendar.model.Event;
+import org.pesmypetcare.webservice.jsonhandler.DateTimeHandler;
+
+import java.util.Arrays;
+
+/**
+ * @author Marc Simó
+ */
+@Data
+public class EventEntity {
+    private String id;
+    private String summary;
+    private String location;
+    private String description;
+    private String color;
+    private Integer emailReminderMinutes;
+    private Integer repetitionInterval;
+    @JsonDeserialize(using = DateTimeHandler.class)
+    private DateTime startDate;
+    @JsonDeserialize(using = DateTimeHandler.class)
+    private DateTime endDate;
+
+    public EventEntity() {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['}' is not preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5403404)